Marlin/src/feature/temp_comp.cpp

```diff
@@ -0,0 +1,288 @@
+#include "temp_comp.h"
```

Comment:
This is not a universal 'temp_com'. It's speciffically for the probe system. (file- and class-names)